Sources/Sentry/SentrySessionReplay.m

```diff
@@ -124,15 +135,25 @@ - (void)captureReplayForEvent:(SentryEvent *)event;
         return;
     }
 
+    [self startFullReplay];
+    [self setEventTag:event];
+
     NSURL *finalPath = [_urlToCache URLByAppendingPathComponent:@"replay.mp4"];
     NSDate *replayStart =
         [_dateProvider.date dateByAddingTimeInterval:-_replayOptions.errorReplayDuration];
 
     [self createAndCapture:finalPath
                   duration:_replayOptions.errorReplayDuration
                  startedAt:replayStart];
+}
 
-    [self startFullReplay];
+- (void)setEventTag:(SentryEvent *)event
+{
+    NSMutableDictionary *tags = @{ @"replayId" : [_sessionReplayId sentryIdString] }.mutableCopy;
```

Comment:
we should plan to remove this tag with proper field. Not working on `context['replay'].replayId`?

Reply:
Correct context name is `contexts['replay'].replay_id`.

Reply:
oh context has `replay_id` and tag has `replayId` ?? 😢